src/main/java/com/lothrazar/cyclic/block/forester/TileForester.java

```diff
@@ -127,9 +128,11 @@ public void tick() {
       }
       else if (this.isSapling(dropMe)) {
         //plant me  . if im on the lowest level
-        if (targetPos.getY() == this.pos.getY()) {
+        Block targetBlock = world.getBlockState(targetPos).getBlock();
+        Boolean validTarget = targetBlock == Blocks.AIR || targetBlock == Blocks.GRASS || targetBlock == Blocks.TALL_GRASS;
```

Comment:
I dont think we need to force-check for exactly GRASS. many saplings can be placed on podzol, or dirt type blocks from mods like Biomes o Plenty.  

The 'rightClickBlock' should validate that this is a valid placement spot .  Looks like when its placing against the gold block, it gets offset to the nearby space.  Similar to how if a player puts a sapling against the side of the block, it still places down. 

I think the problem is here,  the fake player is placing straight ahead.  https://github.com/Lothrazar/Cyclic/blob/95a0e33342dddca48f0a52a76b3befe0ad14757e/src/main/java/com/lothrazar/cyclic/base/TileEntityBase.java#L137

But for the foreseter, we always want to place DOWN, so it hits against the top of the block and validates that way.   

The result==CONSUME looks like a good fix.  I think i might merge this and then update rightClickBlock to take in an optional DIRECTION paramemter to get around this